addons/voice/org.openhab.voice.gtts/build.properties

```diff
@@ -0,0 +1,31 @@
+source.. = src/main/java/
+output.. = target/classes
+bin.includes = META-INF/,\
+               .,\
+	       lib/grpc-core-1.12.0.jar,\
```

Comment:
That's an incredible long list of dependencies... Did you check if all of them are really required for TTS purposes? If here are any that aren't required, it would be nice to reduce the list.

Reply:
It would also be nice when the list is sorted and uses the correct indentation. :-)

Reply:
Unfortunately all are required. I've checked then on by one...
List sorted